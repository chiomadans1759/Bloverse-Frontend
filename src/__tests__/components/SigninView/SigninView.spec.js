import { shallowMount } from '@vue/test-utils'
import SigninView from '@/views/Signin/Signin.vue';

describe('SigninView Component', () => {
  it('should render correctly', () => {
    const wrapper = shallowMount(SigninView);
    expect(wrapper.element).toMatchSnapshot();
  })
});
