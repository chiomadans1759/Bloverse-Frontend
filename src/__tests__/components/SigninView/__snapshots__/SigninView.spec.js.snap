// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninView Component should render correctly 1`] = `
<div
  class="signin-container"
>
  <div
    class="top"
  >
    <router-link
      class="router-link"
      to="/"
    >
      <img
        class="logo"
        src="@/assets/Logo - White.svg"
        style="height: 40px;"
      />
    </router-link>
  </div>
   
  <div
    class="content"
  >
    <signin-component-stub />
  </div>
   
  <div
    class="bottom"
  >
    <p>
      <span
        class="bloverse"
      >
        bloverse
      </span>
       2018 All rights reserved
    </p>
  </div>
</div>
`;
